src/python/robot_wrapper.py

```diff
@@ -22,32 +22,28 @@
 
 class RobotWrapper:
 
-    def __init__(self, filename, mesh_dir = None, root_joint = None):
-        if isinstance(mesh_dir, basestring):
-          build_model_and_geom = True
-        else: # Only load the model
-          build_model_and_geom = False
-          root_joint = mesh_dir
-      
-        self.model_filename = filename
-        if build_model_and_geom:
-          # Check if the module geometry of Pinocchio has been compiled
-          if not "buildModelAndGeomFromUrdf" in dir(se3):
-            raise Exception('It seems that the Geometry Module has not been compiled with Pinocchio') 
-          if(root_joint is None):
-              self.model, self.geometry_model = se3.buildModelAndGeomFromUrdf(filename,mesh_dir)
-          else:
-              self.model, self.geometry_model = se3.buildModelAndGeomFromUrdf(filename,mesh_dir,root_joint)
-
-          self.data = self.model.createData()
-          self.geometry_data = se3.GeometryData(self.data, self.geometry_model)
-        else:
-          if(root_joint is None):
+    def __init__(self, filename, package_dirs = None, root_joint = None):
+        if(root_joint is None):
               self.model = se3.buildModelFromUrdf(filename)
-          else:
+              self.root_added = False
```

Comment:
It was for better clarity of the next lines. Instead of testing once again if root_joint is None to build the geometric model